docs/docs/tutorial/chapter4/authentication.md

```diff
@@ -849,22 +840,16 @@ export const requireAuth = ({ roles }: { roles?: AllowedRoles } = {}) => {
   }
 }
 ```
+</TabItem>
+</Tabs>
 
-:::caution
-
-At this point of the tutorial we have **not added roles** to our user model yet, therefore you can ignore the following error:
-
-`Property 'roles' does not exist on type '{ id: number; email: string; }'.`
+:::info
 
-in the `hasRole` method in `api/src/lib/auth.ts` for now.
+At this point of the tutorial we have **not added roles** to our user model yet, therefore you can ignore the `hasRole` method in `api/src/lib/auth.js` for now.
```

Comment:
Why did you move this out of the TS-only tab? Do you see red squiggles in a JS project too?

Reply:
No, of course not. Neither did i *move this one* out of the tab: i synchronized it with what is written in the JS tab, and then moved both of them below the tab section to reduce pointless duplication.

Sorry, my bad, should have done this in two separate commits, that would have made it clearer.

Reply:
Gotcha, thanks!